driver.go

```diff
@@ -878,13 +878,18 @@ func argToAttr(key string, val interface{}) trace.Attribute {
 	}
 }
 
-func setSpanStatus(span *trace.Span, err error) {
+func setSpanStatus(span *trace.Span, opts TraceOptions, err error) {
 	var status trace.Status
 	switch err {
 	case nil:
 		status.Code = trace.StatusCodeOK
 		span.SetStatus(status)
 		return
+	case driver.ErrSkip:
+		status.Code = trace.StatusCodeUnimplemented
+		if opts.DisableErrSkip {
```

Comment:
Let's add a comment that we are suppressing a non-retryable so perhaps
```go
if opts.DisableErrSkip {
        // Suppress driver.ErrSkip since at runtime some drivers might not have
        // certain features, and an error would pollute many spans.
        status.Code = trace.StatusCodeOK
}
```